sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -269,8 +269,8 @@ def visit_transform(self, transform_node):
               pcoll.element_type, transform_node.full_label)
           key_type, value_type = pcoll.element_type.tuple_types
           if transform_node.outputs:
-            from apache_beam.runners.portability.fn_api_runner_transforms import \
-              only_element
+            from apache_beam.runners.portability.fn_api_runner.transforms \
```

Comment:
You don't have to do this in this PR, but this should probably be in apache_beam/utils.

Reply:
You mean the newly named `translations` module may need to exist in `apache_beam/utils`?

Reply:
I meant that utilities like only_element should probably be in a more common place rather than imported elsewhere from here.